Dockerfile

```diff
@@ -1,7 +1,8 @@
-FROM alpine:3.18
+FROM ghcr.io/stefan-hoeck/idris2-pack:nightly-240625
```

Comment:
Is is possible to use a non-nightly version? And I assume there isn't an official Docker image? I'd prefer us to not use an image that is maintained by a single user. In that case, I'd rather we install Idris ourselves within the docker image.

Reply:
There are only nightly versions of pack, but that is to have available the latest version of packages and have them proven to work together, from what I understand there are very seldom changes in Idris itself or the package manager. Also the [Dockerfile](https://github.com/stefan-hoeck/idris2-pack/blob/main/Dockerfile) is included in the repo and is probably a part of CI for pack. The easiest way I have found to install Idris2 is to actually let pack do it, and if you want to use depend on existing libraries, you kind of want pack. I see no simple option.